samcli/commands/check/graph_context.py

```diff
@@ -0,0 +1,21 @@
+"""
+This class will generate the graph based on the template data.
+Connections between nodes will occur here.
+"""
+from typing import List
+from .resources.Graph import Graph
+
+
+class GraphContext:
```

Comment:
nit: This seems to be better called as GraphFactory or GraphGenerator. Context is already being used for referring to CLI context, such as BuildContext, DeployContext, etc.
Also, can we simply put the generate logic inside of the Graph constructor and eliminate this class entirely?

Reply:
I will put the graph generator logic inside the graph constructor. I agree that is a better place for it.